.github/actions/setup-node/action.yml

```diff
@@ -49,9 +49,9 @@ runs:
         cache: 'yarn'
 
     - name: Set nmMode
-      if: ${{ inputs.hard-link-nm == 'true' }}
+      if: ${{ inputs.hard-link-nm == 'false' }}
       shell: bash
-      run: yarn config set nmMode hardlinks-local
+      run: yarn config set nmMode classic
```

Comment:
you should set it false in the electron build step rather than modify the `actions/setup-node`

Reply:
`nmMode`'s default value in our `.yarnrc` is already `hardlinks-local`, so existing `hard-link-nm == 'true'` in fact does nothing.
https://github.com/toeverything/AFFiNE/blob/29d086652f8b3f02044f47b0ba4f186f1298a414/.yarnrc.yml#L5